### DISCUSSION TITLE
What do you think about hosting your other projects in Github?

### DISCUSSION THREAD

**Discussion by ghost** [RolandHughes/copperspice#5] [General]
CsScintilla, RedDiamond,... They will have more exposure if they are on Github.

**Comment by RolandHughes** [comment]
Don't care. Only reason this project is here is it started as a fork here. When the roadmap is completed and the project gets a new name I may well move it to SourceForge then.

This isn't a project trying to be all things to all people, It is on a direct path to be the OpenSource choice for medical device/systems development.

Gospel truth, when it gets done enough for the rename, one of the "new" features I would like to try adding is Mumps database support. While the Mumps database/language has many different names these days, it's the core of the medical records system for VA hospitals and some commercial medical records systems.

**Comment by RolandHughes** [comment]
> CsScintilla, RedDiamond,... They will have more exposure if they are on Github.

One thing that is unforgivable on Github is this half-baked "releases" thing. On SourceForge we have Files.
https://sourceforge.net/projects/reddiamond/files/

I can park stuff there as it is built. Anyone can pull it down. A Debian might appear a couple of days before an RPM and that a few days before an AppImage. With releases you have to hack up one giant loogie

**Reply by ghost** [reply]
Many projects I know use both SourceForge and Github. SourceForge is for binary releases. Github is for development (hosting the source code, bug reports,...). For example:

https://github.com/adonais/libportable

All binary releases are on SourceForge. Github is only to host the source code and bug reports.

Some projects that decided to stay with SourceForge for a long time, finally make a mirror on Github. This is an example:

https://github.com/DavidNash2024/Win32xx

The Github repository is simply a mirror (but accept bug reports). All development is still done on SourceForge.
